packages/components/src/DataList/components/DataListItem/DataListItem.tsx

```diff
@@ -37,12 +37,19 @@ export function DataListItem<T extends DataListObject>({
 
   const { actions, hasActions } = useGetItemActions<T>(item);
   const isContextMenuVisible = Boolean(contextPosition);
-  const isMultiselectModeActive = Array.isArray(selected) && !!selected.length;
+  const isMultiselectModeActive =
+    Array.isArray(selected) && selected.length > 0;
+  const hoveringSelectedBypass =
```

Comment:
I think it would be nice to provide a more descriptive (self-documenting) name for future references. Perhaps, something like 
```suggestion
  const isSingleRowSelected =
```